cache-filter/src/filter/http.rs

```diff
@@ -16,11 +19,33 @@ use threescale::{
     proxy::cache::{get_application_from_cache, set_application_to_cache},
     structs::*,
 };
-use threescalers::response::{Authorization, UsageReports};
+use threescalers::response::{Authorization, OkAuthorization, UsageReports};
 
 const QUEUE_NAME: &str = "message_queue";
 const VM_ID: &str = "my_vm_id";
 
+#[derive(Debug, Clone, thiserror::Error)]
+enum RateLimitedError {
+    #[error("overflow due to two duration addition")]
+    DurOverflowErr,
```

Comment:
Note that because this is part of an `RateLimitError` enum, it's already clear it is an error, so instead of economizing on the important subject, the Duration, I'd suggest you do so by removing the `Err`/`Error` suffix like this, which leaves no doubt about what has been overflown.
```suggestion
    DurationOverflow,
```

You can also apply this to other error types below.